segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/ThrottlerSettings.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.segmentstore.server.logs;
+
+import com.google.common.base.Preconditions;
+import lombok.Data;
+
+/**
+ * Stores configurable throttling settings to be used in {@link ThrottlerCalculator}.
+ */
+@Data
+class ThrottlerSettings {
+
+    private final int maxBatchingDelayMillis;
+    private final int maxDelayMillis;
+    private final int operationLogMaxSize;
+    private final int operationLogTargetSize;
+
+    public ThrottlerSettings(DurableLogConfig config) {
```

Comment:
Let's use name `ThrottlerConfig` to keep convention and move new configs into it (instead of DurableLogConfig).

Reply:
Throttling is a sub-component of the `DurableLog` and for this reason is configured in that place. The same happens with the metadata checkpoints, as you can see that they are also configured as part of the `DurableLog` and not as a separate component. To follow the current structure, I called the class `ThrottlerPolicy` to be consistent with other classes that do the same, like `MetadataCheckpointPolicy`.